test/features/articles/delete_an_article_test.rb

```diff
@@ -1,16 +1,55 @@
 require "test_helper"
 
 feature "Deleting an Article" do
-  scenario "can delete an existing article" do
-    # Given an existing article (created in the fixtures)
-    sign_in
-    article = articles(:ror)
+  scenario "editors can delete articles" do
+    # Given I am signed in as an editor
+    sign_in(:editor)
 
-    # When I click "destroy"
+    # When I visit the article index page
     visit articles_path
-    click_on('Destroy')
 
-    # Then the article should no longer appear in the list
-    page.text.wont_include(article.title)
+    # Then I can see and click on "Delete" to delete an article
+    page.text.must_include('Delete')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.